tests/test_odm2/test_readservice.py

```diff
@@ -183,8 +185,11 @@ def test_getRelatedModelsByCode(self):
         assert len(resapi) == 0
 
         # test invalid argument
-        resapi = self.reader.getRelatedModels(code = models.ActionBy)
-        assert resapi is None
+        resapi = self.reader.getRelatedModels(code = 234123)
```

Comment:
When I test this on a local postgresql, it gives me a None rather than an empty list. For now, this assertion works, but just an fyi in the future, I have placed a comment in the code.